app/src/main/java/com/android/sample/model/user/User.kt

```diff
@@ -12,7 +12,7 @@ data class User(
     val uid: String,
     val userName: String,
     val profilePictureUrl: String,
-    val fridge: List<Long>,
-    val savedRecipes: List<Long>,
-    val createdRecipes: List<Long>
+    val fridge: List<String>,
```

Comment:
After discussion with @RagdollGoDown, we came to the conclusion that it should be List<String> and not List<Ingredient>. This is mainly because the User ViewModel should search for the Ingredients in the database, and not the class which only updates the user data in the database.